### PR TITLE
Add jsonapi-client 0.9.3 to conda-forge.

### PR DESCRIPTION
**Pull request by sodre**
Includes dependencies:
  - aiohttp
  - async-timeout
  - multidict
  - yarl

### CODE REVIEW

**Comment by sodre** [comment]
The jsonapi-client recipe and dependencies are ready for review.

cc: @conda-forge/core

File: recipes/multidict/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    Multidicts are useful for working with HTTP headers, URL query args etc.
+  doc_url: http://simplejson.readthedocs.io/
+  dev_url: http://github.com/aio-libs/multidict
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - HomerSimpson
+    - LukeSkywalker
```
Comment (isuruf) [inline recipes/multidict/meta.yaml]: Change this

File: recipes/pytest-asyncio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - pytest >=3.0.2
+
+test:
+    imports:
+      - pytest_asyncio
+
+about:
+  home: http://github.com/pytest-dev/pytest-asyncio
+  license: Apache 2.0
+  #license_file: LICENSE # License file is missing in package
```
Comment (isuruf) [inline recipes/pytest-asyncio/meta.yaml]: Can you raise this issue upstream?

Comment (sodre) [inline recipes/pytest-asyncio/meta.yaml]: Issue raised in pytest-dev/pytest-asyncio#49.

File: recipes/multidict/meta.yaml (hunk 1 above)
Comment (sodre) [inline recipes/multidict/meta.yaml]: I will pretend that I was just trying to see if you were paying attention.

Thanks for catching that!

File: recipes/yarl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  source_files:
+    - tests
+  requires:
+    - pytest
+  commands:
+    - py.test tests
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (bgruening) [inline recipes/yarl/meta.yaml]: This looks wrong.

File: recipes/pytest-asyncio/meta.yaml (hunk 2 above)
Comment (sodre) [inline recipes/pytest-asyncio/meta.yaml]: The issue is fixed upstream in their master branch. We have to wait for their next release to add the license file back.

File: recipes/aiohttp/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    - multidict >=2.1.4
+    - async-timeout >=1.2.0
+    - yarl >=0.10.0,<0.11
+
+test:
+  imports:
+    - aiohttp
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (sodre) [inline recipes/aiohttp/meta.yaml]: here it goes again....

File: recipes/jsonapi-client/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    - tests
+  requires:
+    - pytest
+    - pytest-mock
+  commands:
+    - py.test tests
+
+about:
+  home: https://github.com/qvantel/jsonapi-client
+  license: BSD
```
Comment (isuruf) [inline recipes/jsonapi-client/meta.yaml]: Change this to `BSD 3-Clause`

**Comment by isuruf** [comment]
I'm going to merge this. You'll have to restart the CI of the feedstocks because of the dependence of packages here.

**Pull request merged**